dataops_named_entity_recognition/data/data.jsonl

```diff
@@ -0,0 +1,15 @@
+{"text": "The software engineer is working on a new update for the application.", "entity_type": "job title", "results": "software engineer"}
```

Comment:
Think we should avoid recreating this `jsonl` file, since it already exists in

`named_entity_recognition/data/data.jsonl`.